src/current/v23.2/aws-dms.md

```diff
@@ -45,7 +45,9 @@ Complete the following items before starting the DMS migration:
 
     - Drop all [constraints]({% link {{ page.version.version }}/constraints.md %}) per the [AWS DMS best practices](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_BestPractices.html#CHAP_BestPractices.Performance). You can recreate them after the [full load completes](#step-3-verify-the-migration). AWS DMS can create a basic schema, but does not create [indexes]({% link {{ page.version.version }}/indexes.md %}) or constraints such as [foreign keys]({% link {{ page.version.version }}/foreign-key.md %}) and [defaults]({% link {{ page.version.version }}/default-value.md %}).
 
-    - Ensure that any schema changes are also reflected on your tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html). If you make substantial schema changes, the AWS DMS migration may fail.
+    - Ensure that any schema changes are also reflected on your target tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html) to your [table mappings](#step-2-3-table-mappings). If you make substantial schema changes, the AWS DMS migration can fail.
+
+        - [Spatial data types]({% link {{ page.version.version }}/spatial-data-overview.md %}) can have different input and output formats, which may create problems for the DMS migration. To import spatial data columns, you can create a [computed column]({% link {{ page.version.version }}/computed-columns.md %}) on the source database that uses the [`st_asbinary()` function]({% link {{ page.version.version }}/functions-and-operators.md %}#spatial-functions) to convert the spatial type into [Well Known Binary (WKB)]({% link {{ page.version.version }}/well-known-binary.md %}) format. Then use a [transformation rule](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html) to migrate the computed column to a CockroachDB computed column that uses another [spatial function]({% link {{ page.version.version }}/functions-and-operators.md %}#spatial-functions), such as `st_geomfromwkb()`, to convert the WKB values back to the appropriate spatial type.
```

Comment:
Why do we recommend converting to WKB and not EWKB? WKB only support 2D geo types. EWKB can support more dimensions: https://libgeos.org/specifications/wkb/

To do this you'd use `st_asewkb()` instead of `st_asbinary` and `st_geomfromewkb()` instead of `st_geomfromwkb()` in your example.

This may be more of a question for the migrations team. I haven't tested this.

Reply:
I'm not sure if there was a specific reason for recommending WKB over EWKB. @Jeremyyang920 do you think we can find a time to test the workaround?

Reply:
Can see when I have time to test this out, but its probably low in priority on list of things to do. But also wonder how many customers have ran into such an issue before that its worth adding to the docs right now.

Reply:
Understood. In the interest of getting this PR off all our plates, I've amended the guidance to use EWKB (@rharding6373, PTAL) and commented out the guidance so that it won't appear on the site. When we have time to test this workaround, I can open another PR to unhide the guidance.

Reply:
SGTM, thanks for updating.